omas/omas_itm.py

```diff
@@ -355,7 +354,7 @@ def save_omas_itm(ods, user=None, machine=None, shot=None, run=None, new=False,
 # AUTOMATICALLY GENERATED FILE - DO NOT EDIT
 
 def load_omas_itm(user=os.environ['USER'], machine=None, shot=None, run=0, paths=None,
-                   itm_version=default_itm_version, verbose=None):
```

Comment:
@eldond the `omas_itm.py` file is automatically generated from `omas_imas.py`.
You can automatically generate it by running `make itm` in the main omas repository.